### PR TITLE
Flask Web App Templates

### PR DESCRIPTION
**Pull request by VaishnaviNandakumar**
# Pull Request Template

**script name** -
Flask-WebApp-Login Templates

**Brief about script**
Script that creates customizable login and registration templates using Flask and MongoDB for any web app project.

### Self Check(Tick After Making pull Request)

- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in proper folder (Name of folder should be in lowercase with no space in between) (E.g. meet_schedular)
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [x] I have added README.md and requirements.txt (Include version numbers too e.g. pandas==0.0.1) with my script
- [x] Just including required dependencies in requirements.txt (Don't include Python version too)

**If issue was not assigned to you Please don't make a PR. It will marked as invalid.**

### CODE REVIEW

**Comment by VaishnaviNandakumar** [comment]
@pawangeek Hi. How can I resolve the failing check for the Flask app because it's in debug mode?

**Comment by pawangeek** [comment]
Just remove `debug=True` from there

**Review by pawangeek — COMMENTED** [review]
Remove .env file from Root folder

**Comment by VaishnaviNandakumar** [comment]
@pawangeek  The .env file has the sample configurations, is there any other alternative I can use?

**Comment by pawangeek** [comment]
I understand, but put it somewhere in the folder, not in root

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+TEMPLATE=False
```
Comment (pawangeek) [inline .env:1]: I was saying about this file by the way :(

Comment (VaishnaviNandakumar) [reply .env:1]: Oh I missed that. Apologies. Will rectify it right now.

**Comment by pawangeek** [comment]
Perfect, that's fine now :)

**Comment by VaishnaviNandakumar** [comment]
Thank you!

**Pull request merged**